newdle/client/src/components/creation/timeslots/Timeline.module.scss

```diff
@@ -44,8 +45,6 @@ $label-width: 180px;
       .timeline-row-label {
         display: flex;
         justify-content: flex-end;
-        width: $label-width;
-        margin-left: -$label-width;
```

Comment:
you don't have to add this back in the media query below?

Reply:
made changes regarding this part of the css